org.gnome.Keysign.yml

```diff
@@ -0,0 +1,409 @@
+id: org.gnome.Keysign
+branch: master
+runtime: org.gnome.Platform
+runtime-version: '3.28'
+sdk: org.gnome.Sdk
+command: gnome-keysign
+copy-icon: true
+finish-args:
+- --share=ipc
+- --socket=x11
+- --share=network
+- --socket=system-bus
+- --device=all
+- --filesystem=~/.gnupg:ro
+- --filesystem=xdg-run/gnupg:ro
+# We're waiting for webcam support: https://github.com/flatpak/xdg-desktop-portal/issues/38;
+# we use --device=all meanwhile. We do network, because we're opening a port.
+appstream-compose: false
+tags:
```

Comment:
flatpak is only for stable apps, please remove the nightly tag and prefix.

Reply:
Ah that was a leftover. I removed it.